azkaban-common/src/main/java/azkaban/AzkabanCommonModule.java

```diff
@@ -185,4 +176,26 @@ public AzkabanEventReporter createAzkabanEventReporter() {
     }
     return null;
   }
+
+  private void bindImageManagementDependencies() {
+    if(isContainerizedDispatchMethodEnabled()) {
```

Comment:
Please use the negative case first.
if (!isContainer.... ) {
 return;
} 

Rest of the coede 

.....

This way, there is less indentation. Applies to both modules.

Reply:
I believe both are one and the same in this code as we don't have much conditions to handle. So the current code is checking if containerization is enabled then only do the bindings. You are right about the negative usecase and I agree it would be useful if we have more validations or usecases.  But we have only one use case here if containerization is enable then initialize the bindings. So either of the change works. Hence, retaining the same code.

Reply:
We may not have them now and likely never but we never know. However, we can do it later. Approving.